include/deal.II/lac/parpack_solver.h

```diff
@@ -487,6 +515,19 @@ void PArpackSolver<VectorType>::set_shift(const double s )
 
 template <typename VectorType>
 void PArpackSolver<VectorType>::
+set_initial_vector(const VectorType &vec)
+{
+  initial_vector_provided = true;
+  Assert (resid.size() == local_indices.size(),
+          ExcDimensionMismatch(resid.size(),local_indices.size()));
+  vec.extract_subvector_to (local_indices.begin(),
+                            local_indices.end(),
+                            &resid[0]);
```

Comment:
later on i need to give to Parpack wrappers an address where resid starts, that's why resid is `vector<double>` and not `VectorType`.